app/scripts/components/shared/new-button.components.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+class Button extends React.Component {
+	render() {
+		const {
+			size,
+			children,
+			className,
+			...rest
+		} = this.props;
+
+		const classes = classnames(
+			'new-button',
+			{
+				'new-button--tiny': size === 'tiny',
```

Comment:
The goal is to have that "new" button replace the old one so we can remove the prefix.